openedx/core/djangoapps/content/learning_sequences/README.rst

```diff
@@ -0,0 +1,90 @@
+==================
+Learning Sequences
+==================
+
+This package creates a ModuleStore-independent representation of learning
+sequences (a.k.a. "subsections" in Studio), as well as how they are put together
+into courses. It is intended to serve metadata about Learning Sequences to end
+users through the LMS, though it is also available to Studio for pushing data
+into the system. The first API this app implements is computing the Course
+Outline.
+
+---------------
+Direction: Keep
+---------------
+
+This package is being actively developed, but in a very early state. We're not
+going to start feeding data into it (by triggering off of publish) until it's a
+little more complete, so that it's easier to make drastic changes to the data
+model if necessary. During development, you can seed data into it by using the
+``update_course_outline`` management command.
+
+-----
+Usage
+-----
+
+* You may make foreign keys to learning_sequence models from your own app.
+* You may refer to data structures defined in
+  ``openedx.djangoapps.content.learning_sequences.api.data``.
+* Otherwise, you should only ever import and use functions from the top level
+  ``openedx.djangoapps.content.learning_sequences.api`` package. Do not import
+  from anywhere else in the package, including sub-modules of ``api``. There are
+  only a few public methods in
```

Comment:
in...?

This trails off :)

Reply:
Ack, yeah, half-thought. I'll just delete.